docs/changelog.md

```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+3.10.0 (2022-07-01)
+-------------------
+
+**🚀 New features**
+
+* Reorganize and improve map layers settings and attributions
+* Allow to display different layers depending on zoom
+* Allow to define attributions for each layer
+* Allow to define bounds to some layers (#689)
+* Live change attributions to match with the current displayed layer
+* Allow to define a specific layer for offline maps
+* Allow to define different suggestions groups depending on language (#711)
+
+**🐛 Fixes**
+
+* Fix displaying of routes when hovering or clicking on one trek in map search page (#694)
+* Fix first loading of the "Offline contents" page (#621)
+* Fix card display on iOS mobile (#645)
+* Optimize images in the card when there is only one
+* Fix reservation widget loading on treks detail pages (#382)
+
+**💥 Breaking changes**
+
+* Basemap layers settings have been totally modified to allow more customization.  
+* `mapCredits`, `mapClassicLayerUrl`, `mapSatelliteLayerUrl` settings have been replaced by `mapClassicLayers`, `mapSatelliteLayers`, `mapOfflineLayer` and their options.  
+* You have to apply these changes in your `map.json` customization file.
+
+**⚠️ Version notes**
+
+* You have to edit your `map.json` customization file to apply the new layers settings organization. See `map.json` customisation in [documentation](customization.md#settings)
```

Comment:
`customisation` word is okay but it's a british version; we use the american version

Reply:
Yes, indeed. Done.
I don't know why I switched to British style for these lines, while I didn't for others.
Maybe it was too late ! :-)